docs/crds/gslbhostrule.md

```diff
@@ -37,7 +43,7 @@ spec:
 **Note** that site persistence is **disabled** on GslbServices created for **insecure** ingresses/routes, irrespective of this field.
 If this field is not provided in `GSLBHostRule`, the site persistence property will be inherited from the GDP object.
 
-4. `thirdPartyMembers`: To add one or more third party members to a GS from a non-avi site (third party site) for the purpose of maintenance, specify a list of those members. For each member, provide the site name in `site` and IP address in `vip`. Please refer [here](https://avinetworks.com/docs/20.1/gslb-third-party-site-configuration-and-operations/#associating-third-party-services-with-third-party-sites) to see how to add third party sites to existing Gslb configuration. **Note** that, to add third party members, set the `enable` flag in `sitePersistence` to false for this object. If site persistence is enabled for a GSLB Service, third party members can't be added.
+4. `thirdPartyMembers`: To add one or more third party members to a GS from a non-avi site (third party site) for the purpose of maintenance, specify a list of those members. For each member, provide the site name in `site` and IP address in `vip`. Please refer [here](https://avinetworks.com/docs/20.1/gslb-third-party-site-configuration-and-operations/#associating-third-party-services-with-third-party-sites) to see how to add third party sites to existing Gslb configuration. Optional `publicIP` can be added if `vip` IP is private and not accesible by client network .Please check [here](https://avinetworks.com/docs/21.1/nat-aware-public-private-configuration) for more details.   **Note** that, to add third party members, set the `enable` flag in `sitePersistence` to false for this object. If site persistence is enabled for a GSLB Service, third party members can't be added.
```

Comment:
It will be better to change the link to `https://avinetworks.com/docs/latest/nat-aware-public-private-configuration`

Reply:
Changed the link